### PR TITLE
Add redhat-developer/intellij-openshift-connector

### PR DESCRIPTION
**Pull request by jeffmaury**

### CODE REVIEW

**Comment by sbouchet** [comment]
@baijum: can you add me as member on the org please ? i'm part of the devtools team that implement the openshift toolkit for intelliJ. thanks !

**Comment by jeffmaury** [comment]
@baijum any blocker on this. Am I missing something ?

**Comment by baijum** [comment]
> @baijum any blocker on this. Am I missing something ?

Did you tag the right person? I am not working on this project.

**Comment by jeffmaury** [comment]
> > @baijum any blocker on this. Am I missing something ?
> 
> Did you tag the right person? I am not working on this project.

The bot told me to assign it to you. Checking other people in the history

**Comment by jeffmaury** [comment]
@danilo-gemoli Is there something need from my side ? Why is this PR not being processed ?

**Comment by jeffmaury** [comment]
@akashshinde can you review and approve this PR ?

**Comment by jeffmaury** [comment]
@akram can you review and approve this PR ?

File: core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml
Comment (danilo-gemoli) [inline core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml:4]: No `approved` label needed here?

Comment (sbouchet) [reply core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml:4]: @danilo-gemoli : please add if necessary, not sure to understand correctly this coinfig file.

Comment (danilo-gemoli) [reply core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml:4]: It really depends by the workflow you're using. Such a configuration allows Tide to merge PRs as long as they have the `lgtm` label. If you're ok, that's it.

**Comment by akram** [comment]
hi @jeffmaury , I don't think that I am entirely relevant to review this.
Maybe the bot just suggested me as an OWNER or redhat-developer.
To unlock you, I will try to approve it.

/lgtm

**Comment by akram** [comment]
closing it to reopen.

/close

**Review by danilo-gemoli — APPROVED** [review]

**Comment by akram** [comment]
@droslean can you PTAL ?
do you know what is this rehearsals-ack  label ? and why the PR is not authored by openshift-bot ?

**Comment by danilo-gemoli** [comment]
Hi @akram, please have a look [here](https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/#rehearsals). 
[This comment](https://github.com/openshift/release/pull/34118#issuecomment-1318984910) contains the instruction on how to interact with the plugin, just expand the `Interacting with pj-rehearse` section.

File: core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml
Comment (droslean) [reply core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml:4]: This will mean that anyone from the redhat-developer organization can merge a PR by just adding the lgtm. Is this what you need?

Comment (sbouchet) [reply core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml:4]: yes, we want a real person, one of the approver above to merge the PRs

**Comment by sbouchet** [comment]
can't update this PR. please close and comment #36402

**Pull request closed without merging**
